_Group Projects/Group_2/Week_8/program.do

```diff
@@ -0,0 +1,74 @@
+cap program drop pmt 
+clear all
+
+program define pmt, rclass // define an r-class program so that the program writes into return and we can get stuff out of the program
+
+// generate the cutoff for the bottom quintile of 30 observations
+set obs 18 // our intervention has 18 strata 
+gen cluster_id = _n 
+expand 30 // creating 30 duplicates for each cluster 
+bysort cluster_id: gen household_id = _n // creating a household id for each cluster
+
+/* we are generating separate levels of household income based on their strata. 
+For the ease of this simulation, there are broadly three kinds of households. 
+Poor (the first six clusters), middle income (cluster 7-12), and rich (clusters 13-18)
+A similar approach is also adopted for the education and family size variables*/
```

Comment:
Generally, I think this section could use a few more comments, maybe just saying what the variable you're creating is, and what its distribution is/what the range of values end up being (e.g, "create an age variable by taking the absolute value of a normal distribution, where the standard deviation of the distribution varies by household income stratum")

Reply:
Got it, will comment more heavily when I am revising this file!